plugins/lime-plugin-admin/index.js

```diff
@@ -15,5 +15,6 @@ export default {
 		reducer,
 		selector,
 		constants
-	}
+	},
+	isCommunityProtected: true
```

Comment:
would be good to describe the purpose of the default attributes and what the true/false value means.